render/wlr_renderer.c

```diff
@@ -247,9 +249,10 @@ bool wlr_renderer_init_wl_display(struct wlr_renderer *r,
 	return true;
 }
 
-struct wlr_renderer *wlr_renderer_autocreate(EGLenum platform,
-		void *remote_display) {
-	struct wlr_egl *egl = wlr_egl_create(platform, remote_display, NULL);
+struct wlr_renderer *wlr_renderer_autocreate(struct wlr_backend *backend) {
+	int fd = backend_get_drm_fd(backend);
```

Comment:
We need to fail if `fd < 0`.

Reply:
Could it be 0 or is it already checked when initializing it in the backends?

Reply:
0 is a valid FD. For instance, if a compositor closes stdout and then opens a DRM FD, that FD would be assigned the number 0. Yes, it's misleading.

Reply:
The check should be `< 0`, not `== 0`.

Reply:
Oh derp